hawk/app/assets/stylesheets/shared/_table.scss

```diff
@@ -31,5 +31,11 @@
         }
       }
     }
+    td.col-sm-2 {
+        overflow: hidden;
```

Comment:
Line should be indented 6 spaces, but was indented 8 spaces<br>Properties should be ordered max-width, overflow, text-overflow, white-space